assets/css/_vehicle_route_summary.scss

```diff
@@ -69,7 +80,7 @@
   .m-vehicle-route-summary__separator {
     justify-self: center;
     align-self: center;
-    height: 80%;
+    height: 100%;
     background-color: $color-gray-300;
     width: 1px;
   }
```

Comment:
Changes in here were reviewed with design, individual commits labelled with `feedback(design)` have _slightly_ more details.